runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
But why do we even want to "ignore highlights"? Maybe it makes the test look prettier or easier to read, but IMO it is more important that the test tests the _actual screen state_ than that the test looks good. (ref #5197 ) . Also one needs to make decisions (what is important and what is not) for something that should be a rote process, just use `snapshot_util()`, move additional highlights to `set_default_attr_ids`, repeat.

If one wants say that some highlights are "more relevant" or otherwise add information, just add comments to the lines inside `set_default_attr_ids`